Makefile

```diff
@@ -84,6 +93,15 @@ build-darwin:
 container: build
 	docker buildx build --platform="linux/$(ARCH)" --no-cache -t $(IMAGE_TAG) -f Dockerfile --progress=plain .
 
+.PHONY: arc-conformance-container
+arc-conformance-container: docker-buildx-builder
```

Comment:
If we need the e2e test binary always, maybe make `build-e2e-test` a dependency for this target?

Reply:
Ahh. I had it set up that way. But removed to be consistent with other targets